client/components/content-elements/tce-passage/edit/plugins/table-popups/index.js

```diff
@@ -0,0 +1,49 @@
+import get from 'lodash/get';
+
+const EVENT_NAME = 'recalcPositionPopup';
+const NAMESPACE = 'JoditEventDefaultNamespace';
+const SCROLLABLE_PARENT_SELECTOR = '.editor';
+
+export const name = 'TablePopups';
+
+export const install = Jodit => {
+  const { afterInitHook } = Jodit.prototype;
+  Jodit.prototype.afterInitHook = function () {
+    afterInitHook.apply(this, arguments);
+    observeTables(this, Jodit);
+    addScrollHandler(this);
+  };
+};
+
+function observeTables(editor, Jodit) {
+  const $$ = get(Jodit, 'modules.Helpers.$$', () => []);
+  const tableProcessor = get(editor, '__plugins.table', {});
+  if (typeof tableProcessor.observe !== 'function') return;
+  $$('table', editor.editor).forEach(table => {
```

Comment:
Can the `find` method be used here? I assume the observe on the `tableProcessor` will only be called once since there should be only one property on the table object equal to `tableProcessor.__key`. So the code can just find the table it needs and if it exists `observe` it.

Reply:
Every single table that was "given" to the editor needs to be observed so some sort of iteration is needed. `$$` helper (which finds all instances of HtmlTableElement) was only used to avoid any possible side effects because Jodit uses it internally.